kuma/javascript/src/header/dropdown.jsx

```diff
@@ -0,0 +1,93 @@
+//@flow
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+import styled from '@emotion/styled';
+
+const MenuContainer = styled.div`
+    // We use relative here not because we need to tweak positioning
+    // but because we need to position the menu itself relative to the
+    // label position when it is shown.
+    position: relative;
+
+    // We need an explicit pointer-events since we set it to none
+    // on the entire document when the menu is up, but we need to be
+    // able to click on the menu items themselves
+    pointer-events: auto;
+`;
+
+const MenuLabel = styled.div`
+    font-size: 18px;
+    font-weight: bold;
+    padding: 5px 10px;
+    :hover {
```

Comment:
Or maybe the hand cursor to match what is used for links? Actually there is probably an a11y issue here and I should be rendering the menu label as a link or a button or something more semantically correct.

I'll try switching to Button to see if that fixes the cursor issue, and will add a note to the code that there may be an a11y issue, and that maybe the menu items always need to be rendered, even when the menu is closed for the benefit of screen readers.

(I don't want to spend a lot of time on this because I'm hoping that when we do a site redesign the menus just go away!)

Reply:
Switching to a button fixes this.